src/proteomes/config/__tests__/__snapshots__/ProteomesColumnConfiguration.spec.tsx.snap

```diff
@@ -72,9 +72,8 @@ exports[`ProteomesColumnConfiguration component should render column "organism":
 <DocumentFragment>
   <a
     href="/taxonomy/9606"
-    title="Homo sapiens (Human), taxon ID 9606"
   >
-    Homo sapiens (Human)
+    Homo sapiens (Human) (synonym)
```

Comment:
I think that might not be what is expected here

Reply:
Actually nevermind, it's in the mock

Reply:
I know, I did a double take too :-)